notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    os.environ["LATEXIFY"] =str(1)
</pre><p>We set LATEXIFY and FIG_DR only when we want to save figure for our textbook. Otherwise we will not apply latexify in notebooks. Reason is latexify decreases the fig_size and manythigs which may not look good in notebook, so we should use matplotlib default in notebooks. (which you have used in following cells, but this cell is not needed) Again observe this <a href="https://github.com/probml/pyprobml/blob/master/notebooks/book1/02/discrete_prob_dist_plot.ipynb" target="_blank">notebook.</a></p><p> </p><p>We set LATEXIFY and FIG_DR from command line and run notebook from command line using "ipython" as mentioned in this <a href="https://github.com/probml/pyprobml/blob/master/notebooks/README.md" target="_blank">instructions.</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>